src/legacy/core_plugins/vis_type_vega/index.ts

```diff
@@ -24,6 +24,9 @@ import { LegacyPluginApi, LegacyPluginInitializer } from '../../../../src/legacy
 
 const vegaPluginInitializer: LegacyPluginInitializer = ({ Plugin }: LegacyPluginApi) =>
   new Plugin({
+    // TODO: ID property should be changed from 'vega' to 'vis_type_vega'
```

Comment:
@lukeelmers what should we do with it?  
Not sure that we can rename vega.enableExternalUrls

Reply:
Let's leave it as-is for now to prevent breaking things -- we need to figure out a bigger-picture strategy for how we want to deal with changes to configs & saved objects. But this is something that's easy enough to revisit later.

Reply:
Or maybe "easy" wasn't the best word to use...  😛  It's something we can wait on though